compiler/cpp/src/parser/Parser.cpp

```diff
@@ -0,0 +1,196 @@
+//
```

Comment:
I will look at the Parser, Analyzer and Interpreter again in more detail once I've implemented them on my own.

Reply:
Alright 👍